remappings.txt

```diff
@@ -1,6 +1,7 @@
 @openzeppelin/contracts/=lib/openzeppelin-contracts-4.9.3/contracts/
 openzeppelin-contracts-5.0.2/=lib/openzeppelin-contracts-5.0.2/contracts/
 openzeppelin-contracts-upgradeable-4.9.3/=lib/openzeppelin-contracts-upgradeable-4.9.3/contracts/
+openzeppelin-contracts-4.9.3/=lib/openzeppelin-contracts-4.9.3/contracts/
```

Comment:
Is it intentional to need 4.9.3 when we already reference 5.0.2? If yes, i'm curious as to why

Reply:
The problem is @openzeppelin/contracts is referenced in Open Zeppelin upgradeable contracts. At some point, we need to upgrade all contracts to the new version of Open Zeppelin, and at that time change @openzeppelin to point to 5.0.2 (or the latest that will work on our chain at that time).